glean-core/csharp/GleanTests/GleanTests.cs

```diff
@@ -44,5 +44,11 @@ public void GettingUploadEnabledBeforeInitShouldNotCrash()
                 dataDir: tempDataDir
                 );
         }
+
+        [Fact]
+        public void SendAPing()
```

Comment:
Let's remove this test please, since this is incomplete, or add a TODO comment stating that this is incomplete and the whole test needs to be ported over.